scripts/update_version_list_in_security_policy.bash

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+version_major_minor_x="$(cut -f1-2 -d "." version.txt).x"
+
+# skip if version is already in the list
+if ! grep -q "$version_major_minor_x" SECURITY.md; then
+  # prepend new version to the list
+  sed -i "s/white_check_mark:/x:               /g" SECURITY.md
+  sed -i "s/^\\(| -* | -* |\\)$/\\1\\n| $version_major_minor_x  | :white_check_mark: |/" SECURITY.md
```

Comment:
The whitespace for the newline is not perfect, but not too messy and the markdown rendering of GitHub resolves the table correctly regardless of whitespace.